release-notes/2.0/README.md

```diff
@@ -1,10 +1,14 @@
-# .NET Core 2.0 Releases - Out of support
+# .NET Core 2.0 Release -- Out of support
 
-The following .NET Core 2.0 releases have been shipped.
+This release is out of support. See [.NET releases](../../releases.md) for in-support releases.
+
+[.NET Core 2.2](https://blogs.msdn.microsoft.com/dotnet/2018/05/30/announcing-net-core-2-2) was released on December 4th, 2018. The release was a collaborative effort by many [contributors](2.0.0-contributor.md). [.NET Core 2.2 reached end of life on December 23rd, 2019](https://devblogs.microsoft.com/dotnet/net-core-2-2-will-reach-end-of-life-on-december-23-2019/). It was [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](2.2-supported-os.md).
```

Comment:
There are several `2.2` references here.  Are you intending to reference `2.0`?

Reply:
Ooops! Thanks.